apps/explorer/src/explorer/components/common/WithLDProvider/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren } from 'react'
+
+import { withLDProvider } from 'launchdarkly-react-client-sdk'
+import { LAUNCH_DARKLY_CLIENT_KEY } from '@cowprotocol/common-const'
+
+// TODO: remove duplicated component with app/cowswap-frontend/src/modules/application/containers/WithLDProvider
```

Comment:
This is duplicated, I know.

Reply:
I think it's ok, we can keep it as it is